src/components/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,92 @@
+import DatePickerInput from "./DatePickerInput";
+import "./FilterBar.css";
+import { useState } from "react";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Dropdown from 'react-dropdown';
+import 'react-dropdown/style.css';
+import SearchInput from "./SearchInput";
+
+
+const FilterBar = (props) => {
+
+
+  const  optionsCategory = ["Action" 
```

Comment:
Muss jetzt nicht direkt passieren, aber wäre es nicht schlauer die Kategorien aus allen verfügbaren Kategorien auszulesen?

Reply:
wir hatten uns dazu auch gedanken gemacht aber es ist wesentlich einfacher es so zu machen wir sollten darauf achten das wir das ganze nicht zu aufwendig machen irgendwo wir das projekt in einem monat irrelevant sein

Reply:
👍 👍 👍